### PR TITLE
Inko 32 epic reports

### PR DESCRIPTION
**Pull request by mkalema**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by Napho — DISMISSED** [review]
LGTM

File: fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        case
+            when l.loan_type_enum = 1 then 'Individual'
+            when l.loan_type_enum = 2 then 'Group'
+            when l.loan_type_enum = 3 then 'JLG'
+            when l.loan_type_enum = 4 then 'GLIM'
+            when l.loan_type_enum = 5 then 'GSIM'
+        end as 'Loan Type',
+l.term_frequency as 'Terms Duration', l.total_repayment_derived as 'Actual Payment Amount(KES)', l.principal_repaid_derived as 'Principal Paid(KES)', l.interest_repaid_derived as 'Interest Paid(KES)',
+l.fee_charges_repaid_derived as 'Fees Paid(kes)', l.penalty_charges_repaid_derived as 'Total Late Fees Paid(KES)', l.total_overpaid_derived as 'Excess Amount Paid(KES)', l.total_outstanding_derived as 'Current Balance(KES)',
+l.principal_outstanding_derived as 'Principal Balance(KES)', l.interest_outstanding_derived as 'Interest Balance(KES)', l.fee_charges_outstanding_derived as 'Fees Balance(KES)', laa.total_overdue_derived as 'Amount Past Due(KES)',
```
Comment (hero78) [inline fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql]: @mkalema  Is this report  supporting only one currency (KES) ?

**Review by hero78 — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql (hunk 1 above)
Comment (mkalema) [reply fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql]: The excel i got these columns from has these columns as they appear here. I referenced that

Comment (mkalema) [reply fineract-provider/src/main/resources/db/changelog/tenant/commands/insert_portfolio_management_table_reports.sql]: The excel i got these columns from has these columns as they appear here. I referenced that. Temi says we can remove that and add currency as a parameter

**Review by Deepika1095 — APPROVED** [review]

**Review by hero78 — APPROVED** [review]

**Pull request merged**